.github/workflows/release-pr.yml

```diff
@@ -47,6 +47,7 @@ jobs:
   release:
     permissions:
       contents: write
+      pull-requests: write
```

Comment:
note: GITHUB_TOKEN gets read only by default, the NGINX_PAT had write access, so this needs to be granted explicitly